### DISCUSSION TITLE
Mac always get two adb device (even after adb kill-server)

### DISCUSSION THREAD

**Discussion by codehz** [Genymobile/feedback#50] [Genymotion Desktop] [answered]
It make debug thing harder since all adb command will need to specify device id
Host OS: Mac OS 14.2 (Mac Book Air M2)
Genymotion version: 3.4.0 beta 1
adb devices:
```
adb devices -l
List of devices attached
127.0.0.1:5555         device product:motion_phone_arm64 model:Phone device:genymotion transport_id:2
emulator-5554          device product:motion_phone_arm64 model:Phone device:genymotion transport_id:1
```
adb version
```
adb --version
Android Debug Bridge version 1.0.41
Version 34.0.1-9680074
Installed as /Users/***/Library/Android/sdk/platform-tools/adb
```

**Comment by guillaume-roche** [comment, marked as answer]
Hello @codehz 

Thanks for the feedback. This is a known issue on our side, and we hope to fix it for the Genymotion Desktop 3.4.0 finale release.

As a workaround, you can set the default android serial like this:
```
export ANDROID_SERIAL=127.0.0.1:5555
```

Best regards,

**Reply by agavignet** [reply]
This has been fixed in Desktop 3.4
